app/grandchallenge/reader_studies/forms.py

```diff
@@ -142,6 +143,20 @@ class Meta(ReaderStudyCreateForm.Meta):
         }
 
 
+class ReaderStudyCopyForm(Form):
+    copy_images = BooleanField(required=False)
+    copy_hanging_list = BooleanField(required=False)
```

Comment:
We should do sanity checks - it makes no sense to copy the hanging list or case text without the images, etc.

Reply:
Or does it? Unsure.

Reply:
I'd say it makes little sense, a check would be good I think.